flatliners/alertfrequencyversion.py

```diff
@@ -13,7 +13,11 @@ def __init__(self):
         super().__init__()
 
         # holds the state objects for each version and alert
-        self.version_frequencies = dict()
+        # maxsize should be at least equal to the number of unique versions
+        # promql: count (count (cluster_version) by (version)) ,so maxsize = 200
+        # multiple clusters may have the same version, so the version_frequencies
+        # for a specific version should not expire based on time, so ttl=None
```

Comment:
but some versions will go away soon, especially the shorter-lived versions coming from the CI.